src/sagemaker/clarify.py

```diff
@@ -426,68 +426,131 @@ class TextConfig:
     _SUPPORTED_GRANULARITIES = ["token", "sentence", "paragraph"]
     _SUPPORTED_LANGUAGES = [
         "chinese",
+        "zh",
```

Comment:
how are we coming up with this list?

Reply:
These are the [ISO two letter language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) for the supported languages.

Reply:
what I meant is, is this a full list or some of them?

Reply:
This is a full list of supported languages. Not full list of all languages.

Reply:
ok, so are we limited by support by some underlying library or we have some restriction on our side? 
Also whats the plan for keeping them up to date?

Reply:
Yes. We are limited by an underlying library. The update has to be manual.